demos/notebooks/demo_pipeline_cnmfE.ipynb

```diff
@@ -288,17 +443,44 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# compute some summary images (correlation and peak to noise)\n",
-    "cn_filter, pnr = cm.summary_images.correlation_pnr(images[::1], gSig=gSig[0], swap_dim=False) # change swap dim if output looks weird, it is a problem with tiffile\n",
-    "# inspect the summary images and set the parameters\n",
-    "nb_inspect_correlation_pnr(cn_filter, pnr)"
+    "def key_params_cnmfe(cnmfe_model):\n",
+    "    \"\"\"\n",
+    "    Convenience function to return critical parameters given CNMFE estimator object.\n",
+    "    Returns dict with values of rf, stride, gSig, gSiz, merge_threshold, min_corr, min_pnr\n",
+    "    \n",
+    "    Note: \n",
+    "    gSiz is included because it depends on gSig and you want to make sure to change it when you change gSig.\n",
+    "    These are not set in stone: tweak for your own needs!\n",
```

Comment:
"set in stone" might be unclear to ESL.

"tweak for your own needs!" -> "You can explore other values for these parameters."